mlflow/gateway/schemas/chat.py

```diff
@@ -8,7 +8,7 @@
 
 
 class RequestMessage(RequestModel):
-    role: str
+    role: Optional[str]
```

Comment:
Are we making this change purely to support PaLM's API having optional support for role? 
I'm not sure if we want to modify this. With other providers, having this a required field is the guard against a malformed submitted request to their APIs. 
Even though PaLM provides an option to use this or not, is there any downside to making our APIs consistent regardless of provider and keep this as a mandatory field?

Reply:
Yeah so on the Databricks end, we're making it optional only for PaLM requests. I guess we can do something similar here, but would require a bit more reworking. I don't think we should make this mandatory if they can use it without

Reply:
Synced offline with @vladimirk-db, @dbczumar, and Sep, and we decided to enforce the role and keep the API consistent across providers. We can revisit if customers have a use-case